contents/naturalLanguage/nl.md

```diff
@@ -0,0 +1,169 @@
+<frontmatter>
+  title: Introduction to Natural Language Processing (NLP)
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Natural Language Processing (NLP)
+Authors: [Lum Ka Fai Jeffry](https://github.com/j-lum)
+
+* [What is NLP](#what-is-nlp)
+* [Core problems in NLP](#core-problems-in-nlp)
+   * [Syntax](#syntax)
+   * [Semantics](#semantics)
+   * [Relations](#relations)
+* [Applications of NLP](#applications-of-nlp)
+* [What's next](#whats-next)
+* [References](#references)
+
+
+## What is NLP
+Natural Language Processing (NLP) is the set of methods for making human language accessible to computers.
+Recent advances in NLP have given rise to useful tools that have become embedded in our daily lives: 
+spam and phishing classification keeps inboxes sane<sup>[1](#footnote1)</sup>;
+automated chatbots lighten the load on customer support staff and provide customers with immediate feedback<sup>[2](#footnote2)</sup>;
+machine translation bridge the gap between cultures<sup>[3](#footnote3)</sup>. 
+ 
+NLP draws from many other fields of science, from formal linguistics to statistics. 
+The goal of NLP is to provide new computational capabilities around human language: for example, holding a conversation, summarizing an article, and so on.
+
+Even though the study of NLP covers a diverse range of tasks, most of them can be generalized to three themes.
```

Comment:
Might be good to state what the three themes are, i.e. "most of them can be generalized to three themes: Syntax, Semantics and Relations, which are discussed in the next section"